lib/methods/start.js

```diff
@@ -18,19 +19,30 @@ module.exports = async function start() {
     const { devices, modbus } = settings.get();
     const options = {
       baudRate: modbus.baud_rate,
+      parity: modbus.parity ? modbus.parity : 'none',
+      dataBits: modbus.data_bits ? modbus.data_bits : 8,
+      stopBits: modbus.stop_bits ? modbus.stop_bits : 1,
     };
     await this.modbus.connectRTUBuffered(modbus.port, options);
+    this.modbus.setTimeout(1000);
```

Comment:
Shouldn't this be awaited? And potentially a setting? (I know, I also made a PR with a magic number here O:) )